tests/test_field.py

```diff
@@ -111,6 +111,11 @@ def test_returns_a_string_without_placeholders(content):
             {"warning": False},
             ""
         ),
+        (
+            "((warning??This is a conditional warning (with more info in brackets)))",
```

Comment:
Minor: we could also have a test for bracketed text in the middle of the conditional string - this is a great edge case, though 💯.